internal/config/config.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+type Group struct {
+	Orgs []Org `json:"orgs"`
+}
+
+type Groups map[string]Group
+
+type Org struct {
+	ID           int64  `json:"id"`
+	Role         string `json:"role"`
+	GrafanaAdmin *bool  `json:"grafanaAdmin"`
```

Comment:
Nit: Why does this need to be a pointer?

Reply:
answered on the same question done when the implementation happens

Reply:
I am not entirely clear on the answer here either.  It was a bool pointer in the grafana fork, because that is how it is used in their internal code base.  I don't see what a nil check gets us here. If not present in configuration, it defaults to false.

Reply:
This would not default to false from configuration it would default to nil as a pointer default value. If it were a bool it would default to false.

Reply:
Right, that's my point.  Why use a pointer bool, the nil check in this case seems redundant.